dco-signoff-hook/README.md

```diff
@@ -0,0 +1,3 @@
+# Signing off commits
+
+To automatically sign off on every commit, copy the [prepare-commit-msg](prepare-commit-msg) file to the `.git/hooks` directory in your repo or if you already have such a hook, merge the contents into your existing hook.
```

Comment:
Can you also add instructions on how to do this manually? e.g. `git commit -s -m 'This is my commit message'`

Reply:
Similar to https://github.com/kubeflow/website/pull/3682

Reply:
Done.